packages/flutter_markdown/lib/src/builder.dart

```diff
@@ -385,16 +385,31 @@ class MarkdownBuilder implements md.NodeVisitor {
       final _BlockElement current = _blocks.removeLast();
       Widget child;
```

Comment:
Let's move this to just above line 402 so it stays directly attached to the conditionals that initialize it.